build.gradle.kts

```diff
@@ -0,0 +1,145 @@
+import org.gradle.internal.extensions.stdlib.toDefaultLowerCase
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.net.URI
+import java.net.URL
+
+plugins {
+	id("maven-publish")
+	id("fabric-loom") version "1.8.10"
+	id("babric-loom-extension") version "1.8.5"
+}
+
+//noinspection GroovyUnusedAssignment
+java.sourceCompatibility = JavaVersion.VERSION_17
+java.targetCompatibility = JavaVersion.VERSION_17
+
+base.archivesName = project.properties["archives_base_name"] as String
+version = project.properties["mod_version"] as String
+group = project.properties["maven_group"] as String
+
+loom {
+//	accessWidenerPath = file("src/main/resources/examplemod.accesswidener")
+
+	runs {
```

Comment:
Awaik we can specify different folders for each task. Maybe separate client and server tasks into their own folders? (run/client and run/server)